### PR TITLE
Upgrade Django version, install infoNavigator view

### PR DESCRIPTION
**Pull request by CuriousG102**
@rdmurphy

### CODE REVIEW

File: emission_events/emission_events/settings/local.py [hunk 1]
```diff
@@ -24,9 +24,14 @@
 
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 DATABASES = {
-    'default': dj_database_url.config(
-        default=os.environ.get('EMISSIONS_DATABASE')
```
Comment (rdmurphy) [inline emission_events/emission_events/settings/local.py:28]: The current scraper may depend on this, so we probably shouldn't just rip it out.

Comment (CuriousG102) [inline emission_events/emission_events/settings/local.py:28]: I didn't intend to push any changes to local.py. Will revert

**Comment by CuriousG102** [comment]
@rdmurphy ready

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Migrations
+python emission_events/manage.py migrate
+
+# Collect static files
+python emission_events/manage.py collectstatic --noinput
+
+# Run app
+cd emission_events
```
Comment (rdmurphy) [inline docker-entrypoint.sh]: Was it not working without this? It should already be in the proper folder.

Comment (rdmurphy) [inline docker-entrypoint.sh]: Oh, I see what's happening. I'd tack this on to the gunicorn call instead.

`emission_events.emission_events.wsgi:application`

**Pull request merged**